addons/account/models/account_move.py

```diff
@@ -1580,29 +1591,25 @@ def _compute_payments_widget_to_reconcile_info(self):
                 ('parent_state', '=', 'posted'),
                 ('partner_id', '=', move.commercial_partner_id.id),
                 ('reconciled', '=', False),
-                '|', ('amount_residual', '!=', 0.0), ('amount_residual_currency', '!=', 0.0),
```

Comment:
In fact, with the new reconciliation, you can't have one of this field to zero but not the other one. So I think this condition is useless because everything is already filtered by the `reconciled` field.
I could restore this condition if you are afraid of post-migration side effects